protocol/communities_messenger_test.go

```diff
@@ -2401,9 +2413,10 @@ func (s *MessengerCommunitiesSuite) TestRequestAccessWithENSTokenPermission() {
 
 	s.advertiseCommunityTo(community, s.alice)
 
-	requestToJoin := &requests.RequestToJoinCommunity{CommunityID: community.ID()}
+	passwdHash := types.EncodeHex(crypto.Keccak256([]byte(alicePassword)))
+	request := &requests.RequestToJoinCommunity{CommunityID: community.ID(), Password: passwdHash}
 	// We try to join the org
-	response, err = s.alice.RequestToJoinCommunity(requestToJoin)
+	response, err = s.alice.RequestToJoinCommunity(request)
```

Comment:
@0x-r4bbit this test fails, because of:
```
func (n *testNode) NewENSVerifier(_ *zap.Logger) enstypes.ENSVerifier {
	panic("not implemented")
}
```
should i also mock it?


callstack:
```
                /opt/homebrew/Cellar/go/1.20.3/libexec/src/runtime/panic.go:884 +0x204
            github.com/status-im/status-go/protocol.(*testNode).NewENSVerifier(0x1400056dcd8?, 0x1061269fc?)
                /Users/mikhailrogachev/Development/status/status-go/protocol/messenger_test.go:54 +0x2c
            github.com/status-im/status-go/protocol/ens.(*Verifier).ReverseResolve(0x14000db6780, {0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...})
                /Users/mikhailrogachev/Development/status/status-go/protocol/ens/verifier.go:134 +0x34
            github.com/status-im/status-go/protocol/communities.(*Manager).getOwnedENS(0x140005b6000, {0x1400056df60, 0x1, 0x24?})
                /Users/mikhailrogachev/Development/status/status-go/protocol/communities/manager.go:2244 +0x110
            github.com/status-im/status-go/protocol/communities.(*Manager).checkPermissions(0x140005b6000, {0x14000710350?, 0x1, 0x1}, {0x140011463e0, 0x2, 0x140006836c0?}, 0x0)
```

Reply:
Hm.. I don't fully understand how your changes would now trigger this test to fail.
Either way, the test should not fail.

Reply:
added mock for ens